README.md

```diff
@@ -22,9 +22,8 @@ Releases of this project are distributed via clojars, to use it:
   [conn request]
   (log/info "cnc handler got message" request)
   (let [response (-> (message/make-message)
-                     (assoc :targets [(:sender request)]
+                     (assoc :target (:sender request)
                             :message_type "example/cnc_response")
-                     (message/set-expiry 3 :seconds)
                      (message/set-json-data {:response "Hello world"
```

Comment:
I don't thisk this is correct anymore (should be `set-data`). Also, `puppetlabs.pcp.message` isn't in clj-pcp-common 1.0.0.